test/bin/build_images.sh

```diff
@@ -25,6 +25,31 @@ osbuild_logs() {
     done
 }
 
+extract_container_images() {
+    local -r version=$1
+    local -r repo_url=$2
+    local -r outfile=$3
+    local -r repo_name="microshift-extract-images"
+
+    echo "Extracting images from ${version}"
+    mkdir -p "${IMAGEDIR}/release-info-rpms"
+    pushd "${IMAGEDIR}/release-info-rpms"
+    sudo bash -c "cat > /etc/yum.repos.d/${repo_name}.repo" << EOF
+[${repo_name}]
+name="microshift ${version}"
+baseurl="${repo_url}"
+enabled=1
+check_gpg=false
+EOF
+    sudo dnf download microshift-release-info-"${version}" --repo "${repo_name}"
```

Comment:
How about we use `--repofrompath`:
```
$ sudo dnf download --repofrompath test,/home/pm/microshift/_output/test-images/rpm-repos/microshift-local --enablerepo test microshift-release-info
Updating Subscription Management repositories.
Added test repo from /home/pm/microshift/_output/test-images/rpm-repos/microshift-local
test                                                                                                                                                      53 kB/s | 3.7 kB     00:00    

$ ls microshift*                                
microshift-release-info-4.16.0_0.nightly_2024_01_05_154400_20240108094945_d6f4621e0-1.el9.noarch.rpm
```

Reply:
Actually, we can even run it without sudo (added --disablerepo):
```
dnf download --repofrompath test,/home/pm/microshift/_output/test-images/rpm-repos/microshift-local --disablerepo '*' --enablerepo test microshift-release-info
```

Reply:
Done